cglib/src/main/java/net/sf/cglib/core/AbstractClassGenerator.java

```diff
@@ -75,6 +75,9 @@ public Object apply(AbstractClassGenerator gen) {
         };
 
         public ClassLoaderData(ClassLoader classLoader) {
+            if (classLoader == null) {
```

Comment:
Would that not cause a regression when enhancing a system class like `java.lang.Object`?